python/objToJSON.c

```diff
@@ -544,6 +550,20 @@ static void SetupDictIter(HPyContext *ctx, HPy dictObj, TypeContext *pc, JSONObj
   }
 }
 
+static inline int is_type_decimal(HPyContext *ctx, HPy h_obj)
+{
+    int r;
+    if (has_decimal) {
```

Comment:
Maybe change the name of `has_decimal` to `whoever_has_decimal` ? I am not sure I know who `whoever` is: python? json?